aws/resource_aws_rds_cluster.go

```diff
@@ -807,6 +838,10 @@ func flattenAwsRdsClusterResource(d *schema.ResourceData, meta interface{}, dbc
 	d.Set("iam_database_authentication_enabled", dbc.IAMDatabaseAuthenticationEnabled)
 	d.Set("hosted_zone_id", dbc.HostedZoneId)
 
+	if dbc.EnabledCloudwatchLogsExports != nil {
+		d.Set("enabled_cloudwatch_logs_exports", dbc.EnabledCloudwatchLogsExports)
```

Comment:
This is currently failing acceptance testing:

```
=== RUN   TestAccAWSRDSCluster_importBasic
--- FAIL: TestAccAWSRDSCluster_importBasic (112.23s)
    testing.go:518: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=3) {
         (string) (len=33) "enabled_cloudwatch_logs_exports.#": (string) (len=1) "2",
         (string) (len=33) "enabled_cloudwatch_logs_exports.0": (string) (len=5) "audit",
         (string) (len=33) "enabled_cloudwatch_logs_exports.1": (string) (len=5) "error"
        }
        
FAIL
```

We prefer to add error checking when handling aggregate attribute types, which should generate an error here that the types don't match:

```go
if err := d.Set("enabled_cloudwatch_logs_exports", dbc.EnabledCloudwatchLogsExports); err != nil {
  return fmt.Errorf("error setting enabled_cloudwatch_logs_exports: %s", err)
}
```

To fix the error you'll need to convert `dbc.EnabledCloudwatchLogsExports` (a `[]*string`) to `[]string` or `[]interface{}`, which can be accomplished with `flattenStringList()`. This should cover the correct behavior:

```go
if err := d.Set("enabled_cloudwatch_logs_exports", flattenStringList(dbc.EnabledCloudwatchLogsExports)); err != nil {
  return fmt.Errorf("error setting enabled_cloudwatch_logs_exports: %s", err)
}
```